NitroxClient/MonoBehaviours/Multiplayer.cs

```diff
@@ -89,6 +90,11 @@ public void StopCurrentSession()
             remotePlayerManager.RemoveAllPlayers();
```

Comment:
Do we still need to do these things (removeAllPlayers and flush)?  If not, can we remove those code paths?

Reply:
Sure can.

Reply:
Sure can.